### DISCUSSION TITLE
use jsonschema $id w/ cue import --path

### DISCUSSION THREAD

**Discussion by proppy** [cuelang/cue#394] [General]
I'm trying to use bulk `cue import` a collection jsonschema files in the same package, is there a way to pass jsonschema `$id` (or `title`) field as the `--path` flag of the `cue import`command in order to uniqulity identify each schema?

Are the `--path` reference resolved against the jsonschema *before* or *after* its conversion to cue? (it seems that it is the later).

**Comment by mpvl** [comment]
There currently isn't, partly as the attributes are not available from with CUE itself and $id is an attribute.

However, one possibility is to pull the $id into the struct generated for the `--with_context` flag. I'll be all in favor of that.

This should at this time also include the recognized file type.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/394.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
